### PR TITLE
feat: adds wine dataset

### PR DESCRIPTION
**Pull request by amferraboli**
Issue #16 points out the willing of adding more datasets to the lib.

This PR adds the Wine Dataset, a classic toy dataset for ML classification problem. The data and features were collected from https://archive.ics.uci.edu/ml/datasets/Wine and modified to a suitable .csv format.

### CODE REVIEW

File: lib/explorer/datasets.ex [hunk 1]
```diff
@@ -12,4 +12,24 @@ defmodule Explorer.Datasets do
...
+
+    Original Owners:
+    Forina, M. et al, PARVUS -
+    An Extendible Package for Data Exploration, Classification and Correlation.
+    Institute of Pharmaceutical and Food Analysis and Technologies, Via Brigata Salerno,
+    16147 Genoa, Italy.
+
+    Wine. (1991). UCI Machine Learning Repository.
+  """
+  def wine_data,
```
Comment (cigrainger) [inline lib/explorer/datasets.ex:33]:
```suggestion
  def wine,
```

File: priv/datasets/wine_data.csv [hunk 2]
```diff
@@ -0,0 +1,179 @@
+class,alcohol,malic_acid,ash,alcalinity_of_ash,magnesium,total_phenols,flavanoids,nonflavanoid_phenols,proanthocyanins,color_intensity,hue,od280/od315_of_diluted_wines,proline
```
Comment (cigrainger) [inline priv/datasets/wine_data.csv:1]: I'm a bit apprehensive of a `/` in a column name. Might be the R user in me, but it feels destined to break _something_ (even though it does appear to work fine). Perhaps `od280_or_od315_of_diluted_wines`?

**Review by cigrainger — COMMENTED** [review]
Fantastic! Thank you so much for contributing this. Just a few minor things.

**Comment by cigrainger** [comment]
@amferraboli sorry to change things out from under you! With #84 the dataset will live in `datasets/wine.csv` and the function for loading it will match the updated `Explorer.Datasets.fossil_fuels/0`. If you need any help or have any questions don't hesitate to ask!

File: lib/explorer/datasets.ex (hunk 1 above)
Comment (amferraboli) [reply lib/explorer/datasets.ex]: Thanks for the suggestion, adopted!

File: priv/datasets/wine_data.csv (hunk 2 above)
Comment (amferraboli) [reply priv/datasets/wine_data.csv]: After a little research I found out it is actually a ratio to measure protein concentration. So I ended up with `od280_over_od315_of_diluted_wines`. Let me know if you agree.

**Comment by amferraboli** [comment]
@cigrainger thanks for the update. Just made the necessary adjustments to allow merging in the new folder structure and to maintain the loading dataset and naming patterns.

**Pull request merged**